plugins/jenkins/app/models/samson/jenkins.rb

```diff
@@ -250,8 +250,15 @@ def notify_emails
       end
 
       emails.compact!
-      emails.select! { |e| e.include?('@') }
-      emails.map! { |x| Mail::Address.new(x) }
+      emails.map! do |a|
+        begin
+          Mail::Address.new(a)
```

Comment:
do we need to log this situation?

Reply:
did not log it before 🤷 
... also nobody checks the logs anyway and it's easy to reproduce if an email does not get sent since we have all the data